lib/paper_trail.rb

```diff
@@ -190,8 +196,14 @@ def version
 end
 
 # Require frameworks
-if defined?(::Rails) && ActiveRecord::VERSION::STRING >= "3.2"
-  require "paper_trail/frameworks/rails"
+if defined?(::Rails)
+  # Rails module is sometimes defined by gems like rails-html-sanitizer
+  # so we check for presence of Rails.application.
+  if defined?(::Rails.application)
+    require "paper_trail/frameworks/rails"
+  else
+    ::Kernel.warn(E_RAILS_NOT_LOADED)
```

Comment:
Missing namespace, should be: `::Kernel.warn(PaperTrail::E_RAILS_NOT_LOADED)`

Reply:
Good catch, thanks.